### PR TITLE
small updates following async discussion

### PR DESCRIPTION
**Pull request by acholyn**
related to #129 , #128 and #125 as well as updating verbose names to better distinguish `Feedback` and `ProjectNote` models

### CODE REVIEW

File: mod_app/migrations/0026_alter_feedback_options_alter_projectnote_options.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    dependencies = [
+        ("mod_app", "0025_increase_url_length_make_sources_m2m"),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name="feedback",
+            options={
+                "verbose_name": "Feedback on non-academic Activities",
+                "verbose_name_plural": "Feedbacks on non-academic Activities",
```
Comment (tcouch) [inline mod_app/migrations/0026_alter_feedback_options_alter_projectnote_options.py:16]: Feedbacks? I would have thought the plural was just feedback?

File: mod_app/models/film_model.py [hunk 2]
```diff
@@ -72,6 +72,8 @@ def get_absolute_url(self):
         max_length=100, blank=True, null=True
     )  # can use choices if preset
 
+    bfi_identifier = models.CharField(max_length=7, blank=True, null=True)
```
Comment (tcouch) [inline mod_app/models/film_model.py:75]: How sure are you that this is only 7 characters? This wikidata entry for [BFI National Archive work ID](https://www.wikidata.org/wiki/Property:P2703) gives examples with 9 characters.

**Review by tcouch — COMMENTED** [review]
Do ProjectNote and VisualWrittenInfluences need adding to the Admin?

File: mod_app/models/film_model.py (hunk 2 above)
Comment (acholyn) [reply mod_app/models/film_model.py:75]: It's what Aylin said 🤷 : my question followed by her answer in brackets
>BFI identifier – are these always 5 digits or are they of variable length (eg 5-10 digits) and are they always numbers or do they sometimes have letters in as well? (  6-7 numbers)

Comment (tcouch) [reply mod_app/models/film_model.py:75]: Ok. Unless you're relying on the validation, personally I'd opt for allowing longer entries just in case.

**Pull request merged**